build/ansible-compute/group_vars/all/vars.yml

```diff
@@ -0,0 +1,25 @@
+development: true
+autoreduction_user: 'isisautoreduce'
+
+# Database
+db_host: '{{ vault_db_url }}'
+db_username: '{{ vault_db_root_username }}'
+db_password: '{{ vault_db_root_password }}'
+
+# Queue processors
+qp_log_file: 'queueProcessor.log'
```

Comment:
Why is this stored in the vault?

Reply:
It isn't stored in the vault. It is just stored in the global variables file. The vault contains all the keys that look like '{{ vault_**** }}'. You can see that it takes a fixed value (queueProcessor.log), therefore isn't referencing anything in vault.yml.

Reply:
Ah right I follow you, thanks for clarifying.